### PR TITLE
Use connectionFactory's ssl options

### PR DESCRIPTION
**Pull request by WilliamBZA**
Fixes the connection to ssl and AMQPS brokers regression

### CODE REVIEW

File: src/NServiceBus.Transport.RabbitMQ/RabbitMQClusterTransport.cs [hunk 1]
```diff
@@ -24,7 +26,7 @@ public RabbitMQClusterTransport(Topology topology, string connectionString, Queu
         /// <param name="port">The port of the node.</param>
         public void AddClusterNode(string host, int port = 5672)
```
Comment (timbussmann) [inline src/NServiceBus.Transport.RabbitMQ/RabbitMQClusterTransport.cs:27]: could we use `-1` as the default port instead? According to https://rabbitmq.github.io/rabbitmq-dotnet-client/api/RabbitMQ.Client.AmqpTcpEndpoint.html -1 represents the default port, which might be switch correctly depending on the SSL options?

File: src/NServiceBus.Transport.RabbitMQ/RabbitMQClusterTransport.cs [hunk 2]
```diff
@@ -24,7 +26,7 @@ public RabbitMQClusterTransport(Topology topology, string connectionString, Queu
...
         {
-            additionalHostnames.Add($"{host}:{port}");
+            additionalHosts.Add(new Tuple<string, int>(host, port));
```
Comment (timbussmann) [inline src/NServiceBus.Transport.RabbitMQ/RabbitMQClusterTransport.cs:29]: can't we use the simplified `(x,y)` tuple syntax in this project?

Comment (WilliamBZA) [reply src/NServiceBus.Transport.RabbitMQ/RabbitMQClusterTransport.cs:29]: [Computer says no](https://www.youtube.com/watch?v=TcRgyAkLeP4)

File: src/NServiceBus.Transport.RabbitMQ/RabbitMQClusterTransport.cs (hunk 1 above)
Comment (WilliamBZA) [reply src/NServiceBus.Transport.RabbitMQ/RabbitMQClusterTransport.cs]: Updated.

**Review by andreasohlund — APPROVED** [review]

File: src/NServiceBus.Transport.RabbitMQ/RabbitMQClusterTransport.cs (hunk 2 above)
Comment (timbussmann) [reply src/NServiceBus.Transport.RabbitMQ/RabbitMQClusterTransport.cs]: http://gph.is/1UObEqG

**Review by timbussmann — APPROVED** [review]

**Pull request merged**